patches/minecraft/net/minecraft/entity/monster/HoglinEntity.java.patch

```diff
@@ -9,7 +9,15 @@
              this.func_241412_a_(SoundEvents.field_232715_fE_);
              this.func_234360_a_((ServerWorld)this.field_70170_p);
           }
-@@ -212,6 +212,7 @@
+@@ -144,6 +144,7 @@
+          this.func_110148_a(Attributes.field_233823_f_).func_111128_a(6.0D);
+       }
+ 
```

Comment:
This should just call super. There is no reason not to and it makes the patch easier.

Reply:
Done, now just calls super